.github/workflows/android-ci.yml

```diff
@@ -11,55 +11,50 @@ defaults:
     shell: bash -l {0}
 
 jobs:
-  build:
+
+  Build:
+    name: Build
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-
-    - name: Setup gCloud
-      if: github.event.pull_request.head.repo.full_name == github.repository
-      env:
-        BACKPACK_GKEY: ${{ secrets.BACKPACK_GKEY}}
-        GCLOUD_PROJECT_ID: ${{ secrets.GCLOUD_PROJECT_ID }}
-      run: |
-        gpg --quiet --batch --yes --decrypt --passphrase="$BACKPACK_GKEY" --output $HOME/gcloud-service-key.json gcloud-service-key.json.gpg
-        if [ ! -d "$HOME/google-cloud-sdk/bin" ]; then
-          rm -rf $HOME/google-cloud-sdk
-          export CLOUDSDK_CORE_DISABLE_PROMPTS=1
-          curl https://sdk.cloud.google.com | bash
-        fi
-
-        gcloud version
-        gcloud auth activate-service-account --project $GCLOUD_PROJECT_ID --key-file $HOME/gcloud-service-key.json
-    
-    # We install the NDK as the app requires an older version than bundled in the image
-    - name: Install NDK
```

Comment:
is this no longer needed?

Reply:
Not anymore 😄 seems to be fine without it as the bundled version in the VM has it now so we don't need to install/override it